### PR TITLE
feat: add test guide

### PR DESCRIPTION
**Pull request by fabian4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,12 +22,26 @@
...
+1. 下载安装调试客户端。
+
+   > 需要一部非 IOS 系统的手机
+
+   下载并且安装 👉 [终端测试消息APP](https://www.5g-msg.com/#/bussinessInformation) 。
+
+2. 打开 app ，将其设置为系统的默认信息应用。
+3. 关闭wifi，确保使用手机流量上网。
+4. 右上角 -> 设置 -> 常规 -> 融合通信登录。
+5. 向康嘉老师申请账号密码。
```
Comment (su-chang) [inline README.md:42]: 获取康嘉老师许可后，可以在这里附带对应联系方式。否则只有一个名字在这里，作用不大。

**Comment by fabian4** [comment]
ping @huan @IronConrad @lijiarui

**Review by huan — APPROVED** [review]
LGTM

And please follow the suggestion from @su-chang

**Comment by lijiarui** [comment]
> ping @huan @IronConrad @lijiarui

You should get your mentor @su-chang  approval first.

**Review by su-chang — APPROVED** [review]

**Review by IronConrad — APPROVED** [review]

**Pull request merged**